Lib/calendar.py

```diff
@@ -41,6 +42,18 @@ def __str__(self):
         return "bad weekday number %r; must be 0 (Monday) to 6 (Sunday)" % self.weekday
 
 
+def __getattr__(name):
+    if name in ['January','February']:
+        warnings.warn(f"The '{name}' attribute is going to be deprecated use '{name.upper()}' instead",
+                        DeprecationWarning,
+                        stacklevel=2)
+        if name == 'January':
+            return JANUARY
+        else:
+            return FEBRUARY
```

Comment:
Rather than changing the interface and returning the new enums, let's retain returning ints constants as before https://github.com/python/cpython/pull/103642.

This should avoid any surprising int->enum typing changes.